src/controls/richText/RichText.tsx

```diff
@@ -89,6 +89,15 @@ export class RichText extends React.Component<IRichTextProps, IRichTextState> {
     text: strings.ListNumbered,
     data: { icon: 'NumberedList' }
   }];
+  private ddLinkTargetOpts = [{
+    key: '_self',
+    id: 'same',
```

Comment:
I'd just use the same `_self` value for `id`. Don't see a reason to have some custom non-standard name here. Or remove the `id` completely.